src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -217,6 +217,11 @@ function FloatingActionButtonAndPopover(props) {
                         text: props.translate('iou.requestMoney'),
                         onSelected: () => interceptAnonymousUser(() => IOU.startMoneyRequest(CONST.IOU.MONEY_REQUEST_TYPE.REQUEST)),
                     },
+                    {
```

Comment:
Ah good point, `policyExpenseChat` is used for workspace creation, and since we're technically not creating a workspace but only the workspace chat, it should be fine to allow all users to see the `Save the world` option.